packages/go/openapi/src/parameters/header.prefer.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Prefer
+description: Prefer header, used to specify a custom timeout in seconds using the
+  wait parameter as per RFC7240. The default timeout is 30 seconds and maximum timeout
+  is 60 seconds

```

Comment:
Is this still accurate? I see an unused `defaultTimeout` var in middleware.go

Reply:
That's a good callout, with recent changes to timeouts, this may need an update

Reply:
fixed!